### ISSUE TITLE
Best ways to add additional fields to Devise users?

### ISSUE THREAD

**Issue by timfoley** [ga-wdi-exercises/project2#703]
I want to have a `name` field in addition to email and password. I've updated the form, now I need to update some controllers, I believe. 

Found [this guide](http://www.peoplecancode.com/tutorials/adding-custom-fields-to-devise). Anything else you'd suggest I look at/look into?

**Comment by jshawl** [comment]
commit!

**Comment by jshawl** [comment]
also another thing that has helped me in the past is not modifying the devise user model, but creating a completely separate model, e.g. a user has one profile

**Comment by timfoley** [comment]
Aha! That sounds like a good idea to keep in mind. 

I just tried it out using the guide I linked and it's working now that I got around the error noted [here](http://stackoverflow.com/questions/38448598/devise-parameter-sanitizer-for-method-not-found-rails-5) (had to do with deprecated Rails 4 stuff).  

I just need to commit and not be afraid to try things out before posting about them at this point :P.
